### ISSUE TITLE
libsass build problem

### ISSUE THREAD

**Issue by NicoHood** [NicoHood/arc-theme#151]
From AUR:
https://aur.archlinux.org/pkgbase/arc-gtk-theme-git/

libsass was updated yesterday.
Since then it is not possible to build this package anymore. Fails at
package_arc-gtk-theme-git with

```
sassc ./cinnamon/sass/cinnamon.scss ./cinnamon/cinnamon.css;
sassc: error while loading shared libraries: libsass.so.0: cannot open
shared object file: No such file or directory
```

**Comment by fossfreedom** [comment]
Sounds like a specific issue with Arch. Sassc probably need a recompiling with a new version of libsass

**Comment by jnsh** [comment]
> Sounds like a specific issue with Arch. Sassc probably need a recompiling with a new version of libsass

Indeed, this wouldn't be in any way our problem, and the broken `sassc` package is already fixed by [rebuild against the new libsass](https://git.archlinux.org/svntogit/community.git/commit/trunk?h=packages/sassc&id=c5cc08f101cb282e5e0f0dc220194e60d33599d3) in `sassc-3.5.0-2`.

**Comment by NicoHood** [comment]
solved with new sassc 3.5.0-2 which bumped the soversion of libsass.so from 0 to 1
